apps/web/playwright/teams.e2e.ts

```diff
@@ -147,8 +147,13 @@ testBothFutureAndLegacyRoutes.describe("Teams - NonOrg", (routeVariant) => {
     await expect(page.locator(`[data-testid="attendee-name-${testName}"]`)).toHaveText(testName);
 
     // The title of the booking
-    const BookingTitle = `${teamEventTitle} between ${team.name} and ${testName}`;
-    await expect(page.locator("[data-testid=booking-title]")).toHaveText(BookingTitle);
+    const bookingTitle = await page.getByTestId("booking-title").textContent();
+    expect(
+      teamMatesObj?.some((teamMate) => {
+        const BookingTitle = `${teamEventTitle} between ${teamMate.name} and ${testName}`;
+        return BookingTitle === bookingTitle;
+      })
+    ).toBe(true);
```

Comment:
@Amit91848 this is causing the test to be flaky. Sometimes it expects `pro-user` instead of a team mate. and `pro-user` is not expected in the test.

```
expected: Team Event - 30min-team-id-277 between teammate-1 and Test Testson
received: Team Event - 30min-team-id-277 between pro-user and Test Testson
```

cc @Udit-takkar can you take a look as well? Since you have and approval here.

Reply:
This is fixed with #15249 right? @zomars